Scripts/runcts

```diff
@@ -99,7 +99,7 @@ export METAL_ERROR_MODE=3
 export METAL_DEBUG_ERROR_MODE=3
 
 # ----- MoltenVK config settings ------
-export MVK_CONFIG_LOG_LEVEL=0
+export MVK_CONFIG_LOG_LEVEL=1
```

Comment:
Looks like an extraneous change slipped in here. I guess it could be useful to see the logs to figure out why a test fails.

Reply:
NP. Thanks for noticing. But this was intentional...but unrelated. See last line in second commit.

Just included it because I have it set during all my testing anyway.